src/main/java/org/traccar/forward/PositionForwarderMqtt.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright 2022 Anton Tananaev (anton@traccar.org)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.traccar.forward;
+
+import org.traccar.config.Config;
+import org.traccar.config.Keys;
+import org.traccar.helper.MqttUtil;
+import org.traccar.model.Position;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.hivemq.client.mqtt.mqtt3.Mqtt3AsyncClient;
+import com.nimbusds.oauth2.sdk.util.StringUtils;
+
+public class PositionForwarderMqtt implements PositionForwarder {
+
+    private final Mqtt3AsyncClient client;
+    private final ObjectMapper objectMapper;
+    private final String topic;
+
+    /*
+     * Device alias to be used when publishing data
+     */
+    private static final String MQTT_ALIAS = "mqtt.alias";
```

Comment:
What is this parameter?

Reply:
Instead of publishing the data only on a single topic for all devices if a device have an alias defined (in device attributes) position for this device will be published on specific topic. This make easy to filter / subscribe to specific device position information

like
/traccar/position/alias

instead of
/traccar/position



You can see on : https://github.com/traccar/traccar/pull/5202/files/34c88d44dcec94ed13b87eae516581602a0d2362#diff-a232b34e7d9106c3ae8030135f5038e844dc0f6550c8f73f78028a396e9d3195R48

Reply:
I thought we agreed on removing configuration parameters for now.

Reply:
You have requested to remove that OPTIONS parameters.

This is the minimum that I use to integrate this with my environment, split positions by an easy to use name.

Reply:
This is not something we do for forwarding now.